### ISSUE TITLE
[HD][Report] Report for Sanitation Workers

### ISSUE THREAD

**Issue by garimadosar5** [avniproject/hasiru-dala#18]
##### Report 1 - 
- Common registration+Common Sanitation worker
##### Report 2 - 
**One report for each type of sanitation worker tied to the 1st report by individual. uuid**
- Desludging Worker                                      
- Community/Public Toilet Cleaner
- Household Sanitation Worker
- Toilet Cleaner in Schools and Offices
- Toilet Cleaner at Railways 
- Sanitation Worker at Lodge/Hotel/Hospital/Commercial Establishment
- Railway Track Cleaner
- Decanting Station/Sewage Treatment Plant (STP) Worker

**Comment by sidsamanvay** [comment]
@garimadosar5, I see Report 1 is already done. What should be the content of the report 2? And could you rephrase the tieing part
> One report for each type of sanitation worker **tied to the 1st report by individual. uuid**

**Comment by garimadosar5** [comment]
Do not require this report any more.
